src/list/actions.js

```diff
@@ -205,12 +205,12 @@ export function removeItem(id) {
   return async (dispatch) => {
     const actionId = nextActionId++;
     dispatch(removeItemStarting(actionId, id));
-
     await browser.runtime.sendMessage({
       type: "remove_item",
       id,
     });
-    dispatch(removeItemCompleted(actionId, id));
+    const interactive = true;
+    dispatch(removeItemCompleted(actionId, id, interactive));
```

Comment:
nit: match the nearly identical calls from `updateItem()` by simply passing 'true` here.